packages/uilib/src/test-files/simple_v8.scd

```diff
@@ -0,0 +1,14 @@
+<SCL version="2007" revision="B" release="4" xmlns:transnetbw="http://localhost:53515/templates/type-designer.template.xslt">
+<Header id="test"/>
+<DataTypeTemplates>
+	<Private>
+		<transnetbw:SubstationType>
+			<transnetbw:VoltageLevelType />
+			<transnetbw:BayType />
+		</transnetbw:SubstationType>
+		<transnetbw:IEDType>
+		<transnetbw:LDeviceType />
+		</transnetbw:IEDType>
+	</Private>
```

Comment:
how about so?
```xml
<Private>
  <tbw:BayType>
     <tbw:IEDTypeReference typeId="ied-type-1" name="protection 1">
     </tbw:IEDTypeReference
  </tbw:BayType>

  <tbw:IEDType id="ied-type-1">
  </tbw:IEDType>
</Private>
```

The types do not have any direct connections. Type can have child `TypeReferences` referencing other types

Reply:
[File has been moved here](https://github.com/sprinteins/oscd-plugins/blob/3ef3328f67e21f2d586cf8181f63f1d9f0026550/packages/plugins/type-designer/src/public/type-designer.simple.scd)